client/api.go

```diff
@@ -1512,8 +1512,8 @@ func (c *Client) SetAccountDelegate(signer *ethereum.SignKeys, delegate common.A
 	return nil
 }
 
-// SetDelegate sends a set delegate transaction, if op == true adds a delegate, deletes a delegate otherwise
-func (c *Client) CollectFaucetTx(from, to *ethereum.SignKeys, amount, identifier uint64, nonce uint32) error {
+// CollectFaucetTx sends a collect faucet transaction
+func (c *Client) CollectFaucet(from, to *ethereum.SignKeys, amount, identifier uint64, nonce uint32) error {
```

Comment:
just a minor nit, this `CollectFaucet` name got out of sync with the comment in the previous line. a tip for next time, in VSCode, you can rename the function with F2 and it will also change the comment (along other references in code, of course)

Reply:
yup! https://github.com/vocdoni/vocdoni-node/pull/523